content/docs/for-buildpack-authors/how-to/write-buildpacks/specify-launch-processes.md

```diff
@@ -7,6 +7,39 @@ One of the benefits of buildpacks is that they are multi-process - an image can
 
 <!--more-->
 
+A `process type` is a named process definition, contributed by a buildpack at build-time and executed by the launcher at run-time.
+Buildpacks declare process types during the build phase by writing entries into `<layers>/launch.toml`.
+
+## Key Points
```

Comment:
This is a *great* translation of the spec into something more consumable by buildpack authors.